client/css/style.css

```diff
@@ -2083,7 +2084,7 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 
 .window#changelog h3 {
 	font-size: 20px;
-	border-bottom: 1px solid currentColor;
+	border-bottom: 1px solid currentcolor;
```

Comment:
The spec defines currentColor https://www.w3.org/TR/css-color-3/#currentcolor

Reply:
That's done by the auto fix option, I didn't do so manually

Reply:
https://github.com/stylelint/stylelint/issues/5863

Reply:
spec version 4 disagrees https://www.w3.org/TR/css-color-4/#currentcolor-color